cwf/gateway/docker/.env

```diff
@@ -10,9 +10,7 @@
 # limitations under the License.
 
 COMPOSE_PROJECT_NAME=cwf
-DOCKER_REGISTRY=cwf_
-DOCKER_USERNAME=
-DOCKER_PASSWORD=
+DOCKER_REGISTRY=cwf-test.artifactory.magmacore.org/
```

Comment:
Let's go directly with prod no? 
also let's use docker.artifactory.magmacore.org

Reply:
the .env is for testing so I figured it might be good to have the test registries here.
The .prod-env do have the docker.artifactory.magmacore.org.